### PR TITLE
Add jalali calendar

### PR DESCRIPTION
**Pull request by mhbahmani**
As you can see in readme, i added jalali calendar and did a little refactoring.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+python-telegram-bot==12.7
+jdatetime
+convert_numbers
```
Comment (unmonoqueteclea) [inline requirements.txt:3]: Thank you for adding the requirements file! 
Could you please pin to specific versions of these dependencies?

File: utils.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+def separate_callback_data(data):
+    return data.split(";")
+
```
Comment (unmonoqueteclea) [inline utils.py:3]: If this is the same for `telegramcalendar.py` and `telegramjcalendar.py`, it is ok if it is in this `utils.py` module. If it is only used by `telegramjcalendar.py` let's move it to that module

File: utils.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+def reformat_persian_date(date: str) -> str:
+    return date\
+        .replace('یکشنبه', 'یک‌شنبه')\
+        .replace('سه شنبه', 'سه‌شنبه')\
+        .replace('پنجشنبه', 'پنج‌شنبه')
```
Comment (unmonoqueteclea) [inline utils.py:9]: As I don't understand persian dates, a docstring explaining the purpose of this function would be great.

**Review by unmonoqueteclea — CHANGES_REQUESTED** [review]
Great, thank you! Only some little changes requested

File: README.md [hunk 4]
```diff
@@ -28,3 +28,4 @@ def inline_handler(bot,update):
 
 ## Demo
 ![](https://github.com/grcanosa/telegram-calendar-keyboard/blob/master/example.gif)
+![](https://github.com/mhbahmani/calendar-telegram/blob/master/jalali-calendar-demo.gif)
```
Comment (unmonoqueteclea) [inline README.md:31]: Actually, we shuld be using a link to this repo, not to external forks. Could you please change it?

**Review by unmonoqueteclea — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (mhbahmani) [reply requirements.txt:3]: My pleasure! :)
Of course.

File: utils.py (hunk 2 above)
Comment (mhbahmani) [reply utils.py:3]: Yes, it's the smae for both calendars. I remove this function from `telegramcalendar.py` and replcae with the one in `utils.py`.

File: utils.py (hunk 3 above)
Comment (mhbahmani) [reply utils.py:9]: This is so unnecessary and it's just for beautifying the output.
In fact with this function, i'm replacing a full space with half space which is a space with half size! This is sth in persian writing rules. I can add some explanation to this function to be more clear for others or just delete it.

Comment (mhbahmani) [reply utils.py]: Any updates?!

**Comment by unmonoqueteclea** [comment]
Sorry, I will review it in a few minutes

**Review by unmonoqueteclea — APPROVED** [review]
Great! Thank you very much!

**Pull request merged**